spec/color_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../part1/color'
+require 'pry'
+
+RSpec.describe Color do
+
+  let(:rgb_color) { described_class.new(13, 21, 34) }
+  let(:another_color) { described_class.new(55, 89, 144) }
+  index = 20
```

Comment:
use let, try to avoid such assignment in rspec

Reply:
if it not working for your case

try let! 

https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let let us let!